TeslaMateAgile/Services/NordpoolService.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Microsoft.Extensions.Options;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using TeslaMateAgile.Data;
+using TeslaMateAgile.Data.Options;
+using TeslaMateAgile.Services.Interfaces;
+
+namespace TeslaMateAgile.Services;
+
+public class NordpoolService : IPriceDataService
+{
+    private readonly HttpClient _client;
+    private readonly NordpoolOptions _options;
+
+    public NordpoolService(HttpClient client, IOptions<NordpoolOptions> options)
+    {
+        _client = client;
+        _options = options.Value;
+    }
+
+    public async Task<IEnumerable<Price>> GetPriceData(DateTimeOffset from, DateTimeOffset to)
+    {
+        var url = "?currency=," + _options.Currency + "&endDate=" + to.UtcDateTime.ToString("dd-MM-yyyy");
```

Comment:
Should we be sending in the from date at some point here? Calculation ranges can span multiple days in some cases

Reply:
I thought this is handled higher up (I believe I saw it when developing the Barry integration; e.g. if it is spanned over two dates it takes the first, calculates its price and only then takes the second date)